app/assets/javascripts/tylium/behaviors.js.coffee

```diff
@@ -212,7 +212,25 @@ document.addEventListener "turbolinks:load", ->
       else
         $(this).find($('[data-behavior~=scroll-wrapper')).removeClass('hidden');
 
-  # Smooth Scrolling
-  $('[data-behavior~=smooth-scroll]').on 'click', ->
-    target = $(this).data('target');
-    $('[data-id~=' + target + ']')[0].scrollIntoView({behavior: "smooth"});
+  # Disable turbolinks for on-page anchor links (prevents page from jumping to top and allows smooth-scrolling)
+  if $('a[href^="#"]').length
```

Comment:
I don't think we need this check. The `each` in the next line will not iterate anyway if there are no elements found.